cli/src/build/credentials-manage.ts

```
@@ -0,0 +1,1615 @@
+
+  pLog.success(`✓ Removed ${row.key} from ${entry.appId} (${row.sourcePlatforms.join(' + ')}).`)
+  return true
+}
+
+function looksLikeJson(value: string): boolean {
+  const trimmed = value.trim()
+  return trimmed.startsWith('{') || trimmed.startsWith('[')
+}
+
+function sanitizeForFilename(value: string): string {
+  return value.replace(/[^a-z0-9_-]/gi, '_').slice(0, 40)
```

ESLint

[error] Unexpected character class ranges '[a-z0-9_]'. Use '\w' instead.

(regexp/prefer-w)